west.yml

```diff
@@ -182,7 +182,7 @@ manifest:
       path: modules/debug/mipi-sys-t
       groups:
         - debug
-      revision: 2f51407b2ac68e2be41af322a42905063f009d75
+      revision: pull/9/head
```

Comment:
Need to remove this here. The west update commit should then be moved before this commit.

Reply:
That is addressed in the next commit in the set, but I can squash them.

Reply:
You can squash them, or reorder so the west.yml change appears beforehand. Using a PR as west module revision is simply for CI to test the whole thing before things are merged in the module repo. Once the changes are in the module repo, west.yml should use SHA instead.